Bugzilla/Elastic/Indexer.pm

```diff
@@ -179,6 +179,7 @@ sub bulk_load {
     my $new_ids     = $self->_select_all_ids($class, $last_id);
     my $updated_ids = $self->_select_updated_ids($class, $last_mtime);
 
+    $self->put_mapping($class);
```

Comment:
This is actually fixing a pretty bad mistake in the old code, that wouldn't update the mapping if we ever deleted the ES index.